MaxText/input_pipeline.py

```diff
@@ -84,6 +88,8 @@ def _normalize_features(features):
 # -----------------------------------------------------------------------------
 
 
+# this method needs a rewrite in pytorch land (assuming no use of tf.data and
+# jax api)
```

Comment:
Most of the pre-process operations like map, filter, padding are easily transferrable as they are available for huggingface datasets as well.